1-js/11-async/01-callbacks/article.md

```diff
@@ -37,7 +37,7 @@ We can use this function like this:
 loadScript('/my/script.js');
 ```
 
-The script is executed "asynchronously", as it starts loading now, but runs later, when the function has already finished.
+The script is executed "asynchronously", as it starts loading now but runs later when the function has already finished.
 
 If there's any code below `loadScript(…)`, it doesn't wait until the script loading finishes.
```

Comment:
I prefer keeping at least the first comma. Second more optional, but I still think it works better with it than without it. Ideas settle in my head better w/ the commas.